engine/execution/computation/manager.go

```diff
@@ -127,6 +128,13 @@ func (e *Manager) ExecuteScript(code []byte, arguments [][]byte, blockHeader *fl
 
 	startedAt := time.Now()
 
+	trackerID := rand.Uint32()
```

Comment:
Could you add a comment here with some context on why we're doing this, and either a TODO to remove this when it is no longer needed, or maybe make it configurable by flag later on.

Reply:
added.